taskcluster/scripts/detect-target.sh

```diff
@@ -64,7 +66,16 @@ guess_target() {
   local DOUBLE=""
   case "$OS" in
     "darwin") DOUBLE="apple-darwin" ;;
-    "linux") DOUBLE="unknown-linux-gnu" ;;
+    "linux")
+      # In the case of linux, we'll use the availability of
+      # a musl build to choose between musl and gnu.
+      DOUBLE="unknown-linux"
+      if [ -d "${SCRIPT_DIR}/../$ARCH-$DOUBLE-musl" ] ; then
+        DOUBLE="$DOUBLE-musl"
+      else
+        DOUBLE="$DOUBLE-gnu"
+      fi
+      ;;
```

Comment:
Is this used outside the megazords? because we aren't building targeting musl there.

Reply:
Oh you're right. Up until this PR we were [building cirrus against musl](https://github.com/mozilla/application-services/blob/main/taskcluster/ci/server-megazord-build/kind.yml#L42-L45). You want me to remove this?

Reply:
Now removed.